src/objects/core/zcl_abapgit_serialize.clas.abap

```diff
@@ -150,12 +153,30 @@ CLASS ZCL_ABAPGIT_SERIALIZE IMPLEMENTATION.
     LOOP AT lt_files INTO ls_file.
       APPEND INITIAL LINE TO ct_files ASSIGNING <ls_return>.
       <ls_return>-file = ls_file.
+
+      zcl_abapgit_file_status=>identify_object(
```

Comment:
the calls to `identify_object` is for handing the namespace escaping? suggest extracting the namespace handing to a new public method, it will make it more clear what happens here

Reply:
just added a comment for now. I will refactor `identify_object` (and the `zcl_abapgit_object_files=>filename`) into a nice `zcl_abapgit_filename_logic` with UTs in separate PR

Reply:
cool 👍